flixel/input/actions/FlxActionInputAnalog.hx

```diff
@@ -0,0 +1,362 @@
+package flixel.input.actions;
+
+import flixel.input.FlxInput;
+import flixel.input.actions.FlxAction;
+import flixel.input.actions.FlxActionInput;
+import flixel.input.gamepad.FlxGamepad;
+import flixel.input.gamepad.FlxGamepadInputID;
+import flixel.input.mouse.FlxMouseButton;
+
+#if steamwrap
+import steamwrap.api.Steam;
```

Comment:
Right, but nowhere in this file specifically are you using those classes. FlxSteamController should have those imports, but not here, and that's what haxe-syntax is catching.